modules/backend/lang/en/lang.php

```diff
@@ -360,7 +360,9 @@
         'no_wrap' => 'Do not wrap tags',
         'no_wrap_comment' => 'The list of tags that should not be wrapped inside block tags.',
         'remove_tags' => 'Remove tags',
-        'remove_tags_comment' => 'The list of tags that are removed together with their content.'
+        'remove_tags_comment' => 'The list of tags that are removed together with their content.',
+        'toolbar_buttons' => 'Toolbar Buttons',
+        'toolbar_buttons_comment' => 'The Toolbar Buttons to be displayed in the Rich Editor. [fullscreen, bold, italic, underline, strikeThrough, subscript, superscript, fontFamily, fontSize, |, color, emoticons, inlineStyle, paragraphStyle, |, paragraphFormat, align, formatOL, formatUL, outdent, indent, quote, insertHR, -, insertLink, insertImage, insertVideo, insertAudio, insertFile, insertTable, undo, redo, clearFormatting, selectAll, html]',
```

Comment:
Please also change `The Toolbar Buttons to be displayed in the Rich Editor` to `The Toolbar Buttons to be displayed in the Rich Editor by default`